miniupnpd/netfilter_nft/nftnlrdr_misc.c

```diff
@@ -440,18 +487,21 @@ parse_rule_cmp(struct nftnl_expr *e, rule_t *r) {
 			r->reg1_type = RULE_REG_NONE;
 			return;
 		}
+		FALL_THROUGH;
```

Comment:
@paul-chambers  Dont you think theyre should be "break;" instead of "FALL_THROUGH" in this switch/case ?

Reply:
That was my first thought when I went to look at what was causing the warnings. My reasoning was that the original author was either a) poorly skilled and didn't test their code, or b) they intentionally wanted to fall through, just didn't know how to suppress the warnings (or didn't care about them).

I admit I didn't go through the existing code to figure out whether those should have been breaks. Since I haven't gotten nftables support working yet, I'm not able to test a change I make in other people's code, so I took the safe route, assumed the person knew what they were doing, and suppressed the warnings.

Reply:
> I don't understand why there is a merge in your Pull request...
> I'm going to rebase

That was me making sure I was current with your repo.

I didn't understand @2pac8831's comments, so I went through things again, looking for things I may have missed. Turns out he was wasting my time. Sorry for any confusion.